lib/controllers/backend/spree/admin/user_passwords_controller.rb

```diff
@@ -9,6 +9,8 @@ class Spree::Admin::UserPasswordsController < Devise::PasswordsController
   helper 'spree/admin/tables'
   layout 'spree/layouts/admin'
 
+  skip_before_action :require_no_authentication, only: [:create]
```

Comment:
The route is to send the reset email which is open to people without authentication. For example, you aren't logged in when you request a recovery email. Skipping this action allows logged in admins to request the recovery email.